### PR TITLE
Laptops database and tables with some styling

### PR DESCRIPTION
**Pull request by danielsteenberg**
Created laptop database and table. Did some styling on the forms and display info

### CODE REVIEW

File: app/assets/stylesheets/devise-edit.scss [hunk 1]
```diff
@@ -22,7 +22,7 @@ h2 {
 }
 
 input {
-  background: linear-gradient(black, grey);
+  background-color: grey;
   color: white;
```
Comment (jamieandyou) [inline app/assets/stylesheets/devise-edit.scss]: You don't have to change this now. But try get into the habit of using hex codes for your colours.

Example:
Use #808080 for grey
And #fff for white
etc.

File: app/assets/stylesheets/inputform.scss [hunk 2]
```diff
@@ -1,11 +1,22 @@
 .form {
-	border: solid #000000;
-	width: 50%;
-	padding: 40px;
+	border-radius: 10px;
```
Comment (jamieandyou) [inline app/assets/stylesheets/inputform.scss]: Indentation

File: app/views/laptops/index.html.slim [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    = link_to 'New Laptop', new_laptop_path, class: 'btn btn-primary'
+
+  .float-right
+    h2 Search Bar
+    form.search-form
+      | Enter Search item here:
+      br/
+      input name="username" type="text" /
+      br/
+      input.button onclick="alert('Search')" type="button" value="Search" /
```
Comment (jamieandyou) [inline app/views/laptops/index.html.slim]: You can take the / off the end of all the above lines

File: app/views/laptops/new.html.slim [hunk 4]
```diff
@@ -0,0 +1,7 @@
+.container
+  h1 New laptop
+
+  .form
+    = render 'form'
+
+   	= link_to 'Back', laptops_path, class: 'btn btn-primary'
```
Comment (jamieandyou) [inline app/views/laptops/new.html.slim]: If the link_to back is supposed to be underneath and separate to the form, then the = link_to must have the same indentation level as the above = render 'form' line.
It mustn't be indented more.

File: app/models/laptop.rb [hunk 5]
```diff
@@ -0,0 +1,3 @@
+class Laptop < ActiveRecord::Base
+	years = ['2012', '2013', '2014']
```
Comment (jamieandyou) [inline app/models/laptop.rb]: What are you using this for?

File: app/assets/stylesheets/devise-edit.scss (hunk 1 above)
Comment (danielsteenberg) [inline app/assets/stylesheets/devise-edit.scss]: sorted

File: app/views/laptops/index.html.slim (hunk 3 above)
Comment (danielsteenberg) [inline app/views/laptops/index.html.slim]: done

File: app/views/laptops/new.html.slim (hunk 4 above)
Comment (danielsteenberg) [inline app/views/laptops/new.html.slim]: sorted

File: app/models/laptop.rb (hunk 5 above)
Comment (danielsteenberg) [inline app/models/laptop.rb]: Wanted to create a dropdown for selecting the year in laptops. Deleting it for now

File: app/assets/stylesheets/devise-edit.scss [hunk 6]
```diff
@@ -22,8 +22,8 @@ h2 {
 }
 
 input {
-  background: linear-gradient(black, grey);
-  color: white;
+  background-color: #808080;
+  color: #ffffff;
```
Comment (jamieandyou) [inline app/assets/stylesheets/devise-edit.scss]: Rather just use #fff
It's cleaner than the long version

File: app/views/employees/new.html.slim [hunk 7]
```diff
@@ -3,5 +3,4 @@
 
 	.form
 	  = render 'form'
-		
-	= link_to 'Back', employees_path, class: 'btn btn-primary'
+		= link_to 'Back', employees_path, class: 'btn btn-primary'
```
Comment (jamieandyou) [inline app/views/employees/new.html.slim]: This should be the same indentation as the = render 'form' above it

File: app/views/laptops/new.html.slim (hunk 4 above)
Comment (jamieandyou) [inline app/views/laptops/new.html.slim]: Doesn't seem sorted

File: app/assets/stylesheets/devise-edit.scss (hunk 6 above)
Comment (danielsteenberg) [inline app/assets/stylesheets/devise-edit.scss]: sorted

File: app/views/employees/new.html.slim (hunk 7 above)
Comment (danielsteenberg) [inline app/views/employees/new.html.slim]: sorted

Comment (jamieandyou) [inline app/views/employees/new.html.slim]: One last place, Dan :)

**Comment by jamieandyou** [comment]
:+1:

**Pull request merged**